pages/common/trash-cli.md

```diff
@@ -0,0 +1,23 @@
+# trash-cli
+
+> Trash-cli is a less permanent way of removing files via the command line.
+
+- Trash files and directories:
+
+`trash-put {{file_name}}`
+
+- Empty the trashcan(s):
```

Comment:
So does this mean there can be multiple trashcans ? How do we control which trash can to dump a file to ?

Reply:
No, I'll get rid of that